locales/en/index.yml

```diff
@@ -996,6 +997,15 @@ btnUpdateApp: Update the IO app
 systemsOff:
   title: Unable to start IO
   closeApp: close the application and try again
+instabug:
+  contextualHelp:
+    title1: Do you need help?
+    buttonChat: Send us a message
+    descriptionChat: You can use our chat tool to send a message in case of urgent need for assistance. You will receive an answer directly on IO
+    buttonBug: Report a bug
+    descriptionBug: You can report an error or something that does not work to our developers, they will take care of the problem as soon as possible.
+    title2: Are you new to IO?
+    descriptionLink: Find out how it works and what are the special features of IO, by visiting the site
```

Comment:
It seems to be never used

Reply:
it is used in the new component `InstabugAssistanceComponent`

Reply:
unused strings removed within https://github.com/pagopa/io-app/pull/1629/commits/da1bbdfd04c40079d66e00455e3705a3b9a5b0c9